azure-pipelines.yml

```diff
@@ -201,6 +201,7 @@ jobs:
         MATPLOTLIB_VERSION: 'min'
         THREADPOOLCTL_VERSION: '2.2.0'
         SKLEARN_ENABLE_DEBUG_CYTHON_DIRECTIVES: '1'
+        SKLEARN_RUN_FLOAT32: '1'
```

Comment:
Let's make it explicit that this new environment variable only impacts the tests:

```suggestion
        SKLEARN_TESTS_RUN_FLOAT32: '1'
```